stemcell_builder/stages/bosh_config_root_ssh_login/apply.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2009-2012 VMware, Inc.
+
+set -e
+
+base_dir=$(readlink -nf $(dirname $0)/../..)
+source $base_dir/lib/prelude_apply.bash
+
+sed "/^ *PermitRootLogin/d" -i $chroot/etc/ssh/sshd_config
+echo 'PermitRootLogin yes' >> $chroot/etc/ssh/sshd_config
+
+sed "/^ *AllowGroups/d" -i $chroot/etc/ssh/sshd_config
+#echo 'AllowGroups bosh_sshers' >> $chroot/etc/ssh/sshd_config
+
+sed "/^ *DenyUsers/d" -i $chroot/etc/ssh/sshd_config
+#echo 'DenyUsers root' >> $chroot/etc/ssh/sshd_config
```

Comment:
Is there a reason to leave the commented out lines? Also can you please add a newline at the end of this file?